### ISSUE TITLE
Build fails because of coreos/etcd checksum mismatch

### ISSUE THREAD

**Issue by skitt** [operator-framework/operator-lifecycle-manager#1032]
## Bug Report

**What did you do?**

I cloned the OLM repository, and tried running `make` inside it.

**What did you expect to see?**

I expected the tests to pass and the build to succeed.

**What did you see instead? Under which circumstances?**

```
go: downloading github.com/coreos/etcd v3.3.10+incompatible
go: downloading k8s.io/kubernetes v1.14.5-beta.0.0.20190708100021-7936da50c68f
go: downloading google.golang.org/genproto v0.0.0-20181016170114-94acd270e44e
go: downloading github.com/go-openapi/validate v0.19.2
go: extracting github.com/go-openapi/validate v0.19.2
go: downloading github.com/go-openapi/loads v0.19.2
go: extracting google.golang.org/genproto v0.0.0-20181016170114-94acd270e44e
go: downloading github.com/go-openapi/runtime v0.19.0
go: extracting github.com/go-openapi/loads v0.19.2
go: downloading github.com/go-openapi/analysis v0.19.2
verifying github.com/coreos/etcd@v3.3.10+incompatible: checksum mismatch
	downloaded: h1:jFneRYjIvLMLhDLCzuTuU4rSJUjRplcJQ7pD7MnhC04=
	go.sum:     h1:KjVWqrZ5U0wa3CxY2AxlH6/UcB+PK2td1DcsYhA+HRs=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
go test  -v -race -tags=json1 -coverprofile=cover.out -covermode=atomic \
	-coverpkg ./pkg/controller/... ./pkg/...
go: downloading k8s.io/kubernetes v1.14.5-beta.0.0.20190708100021-7936da50c68f
go: downloading github.com/coreos/etcd v3.3.10+incompatible
go: downloading github.com/mattn/go-sqlite3 v1.10.0
go: downloading github.com/go-openapi/runtime v0.19.0
go: extracting github.com/go-openapi/runtime v0.19.0
go: downloading github.com/go-openapi/analysis v0.19.2
go: extracting github.com/go-openapi/analysis v0.19.2
verifying github.com/coreos/etcd@v3.3.10+incompatible: checksum mismatch
	downloaded: h1:jFneRYjIvLMLhDLCzuTuU4rSJUjRplcJQ7pD7MnhC04=
	go.sum:     h1:KjVWqrZ5U0wa3CxY2AxlH6/UcB+PK2td1DcsYhA+HRs=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
make: *** [Makefile:42: cover.out] Error 1
```

**Environment**
* operator-lifecycle-manager version:

d6056ddf181798e740178d2c6cad76d60bd0b52c

* Kubernetes version information:

```
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.5", GitCommit:"51dd616cdd25d6ee22c83a858773b607328a18ec", GitTreeState:"archive", BuildDate:"2019-03-05T13:36:18Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.0", GitCommit:"641856db18352033a0d96dbc99153fa3b27298e5", GitTreeState:"clean", BuildDate:"2019-03-25T15:45:25Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
```

* Kubernetes cluster kind: 

KIND

**Possible Solution**

Updating the checksum in `go.mod`, but I haven’t figured out why it’s changed...

**Additional context**

N/A.

**Comment by ecordell** [comment]
I'm not sure why you're seeing  a checksum mismatch,  deleting go.sum and revendoring gives me the same checksum that's in master - but I suspect your environment is not using go modules, which we rely on for building.

What version of `go` are you using locally? Our make scripts switch to vendoring for 1.11 and 1.12, so if you're on 1.13 we just need to change our makefile.

**Comment by skitt** [comment]
> I'm not sure why you're seeing a checksum mismatch, deleting go.sum and revendoring gives me the same checksum that's in master - but I suspect your environment is not using go modules, which we rely on for building.
> 
> What version of `go` are you using locally? Our make scripts switch to vendoring for 1.11 and 1.12, so if you're on 1.13 we just need to change our makefile.

I ran into this with 1.11, and upgraded to 1.13 to get its improved module support. #1038 fixes the Makefile, I’ll check again what happens with 1.11...

It’s still weird that the checksum changed, or is that expected with the switch to modules? There are a number of other projects which still have the old checksum in their `go.sum`.

**Comment by skitt** [comment]
> I ran into this with 1.11, and upgraded to 1.13 to get its improved module support. #1038 fixes the Makefile, I’ll check again what happens with 1.11...

I can’t reproduce this with 1.11, and #1038 fixes the problem on 1.13.
